README.md

```diff
@@ -4,27 +4,29 @@ Today we're going to build a clone of a social network (really similar to the bl
 
 As any other social network we're going to have different types of `Post`s. All these different types of Posts should inherit from the parent `Post` class:
 
-* `TextPost`: Just a simple text post. Should be constructed as `TextPost(text="Post Text").
-* `PicturePost`: A post containing text and the URL of a picture: Should be constructed as `PicturePost(text="Post Text").
+* `TextPost`: Just a simple text post. Should be constructed as `TextPost(text="Post Text")`.
+* `PicturePost`: A post containing text and the URL of a picture: Should be constructed as `PicturePost(text="Post Text", image_url="imgur.com/OAWTSJu")`.
 * `CheckInPost`: A post containing text and coordinates of the user's position. Should be constructed as `CheckInPost(text="Post Text", latitude="40.741895", longitude="-73.989308")`.
 
-Rmotrgram also has users. A user is a simple class that can be created as: `User(first_name, last_name, email)`.
+Rmotrgram also has users. A user is a simple class that can be created as: `User(first_name="John", last_name="Smith", email="johnsmith@rmotr.com")`. 
```

Comment:
Not sure which approach is better in this case. Is either pattern of description established in earlier projects? If this were PYP I'd say definitely the original version, but since it's ITP not really sure.